core/arch/arm/tee/entry_fast.c

```diff
@@ -164,6 +175,11 @@ void tee_entry_fast(struct thread_smc_args *args)
 	case OPTEE_SMC_ENABLE_SHM_CACHE:
 		tee_entry_enable_shm_cache(args);
 		break;
+#ifdef CFG_BOOT_SECONDARY_REQUEST
```

Comment:
remove the `#ifdef` here. function`tee_entry_boot_secondary()` shall be define in both cases.
see comment above.